CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 ## 🧰 Bug fixes 🧰
 
 ## 💡 Enhancements 💡
-
+- Clarifying expectations with regards to adding a component to the collector contrib
```

Comment:
This is a documentation update not a functional enhancement. So it shouldn't be listed here

Reply:
However, it does impact developers whom are contributing to it, so I have added it in for that reason.

Reply:
More context on the other PR I had been working on to help clarify when updates are needed to be made to the changelog.

https://github.com/open-telemetry/opentelemetry-collector/pull/4491